packages/esm-service-queues-app/src/queue-table/cells/queue-table-action-cell.component.tsx

```diff
@@ -23,7 +23,7 @@ export function QueueTableActionCell({ queueEntry }: QueueTableCellComponentProp
         size={isDesktop(layout) ? 'sm' : 'lg'}>
         {t('transition', 'Transition')}
       </Button>
-      <OverflowMenu aria-label="Actions menu" size={isDesktop(layout) ? 'sm' : 'lg'} flipped>
+      <OverflowMenu aria-label="Actions menu" size={isDesktop(layout) ? 'sm' : 'lg'} align="left" flipped>
```

Comment:
minor cosmetic fix to prevent the "Options" tooltip from expanding the table width on hover, which makes the horizontal scrollbar appear

![image](https://github.com/user-attachments/assets/fc0b20e1-4872-4231-ad76-97aa5028a129)